HabitTracker.BatataDoc3/Program.cs

```diff
@@ -0,0 +1,27 @@
+﻿using HabitTracker.BatataDoc3.db;
+using HabitTracker.BatataDoc3.HabitTracker;
+
+namespace HabitTracker.BatataDoc3
+{
+    internal class Program
+    {
+        static void Main()
+        {
+
+            Crud crud = new Crud();
+            bool newDb = crud.StartDatabase();
+
+            Console.WriteLine(@"========================
+WELCOME TO HABIT TRACKER
+========================");
+            HabitTrackerApp ht = new HabitTrackerApp(crud);
```

Comment:
Very bad variable name. The following are better:
`var habitTracker = new HabitTrackerApp(crud);`
`var app = new HabitTrackerApp(crud);`